front/app/modules/commercial/moderation/admin/containers/index.tsx

```diff
@@ -421,9 +387,10 @@ const Moderation = () => {
     if (!processing) {
       setSelectedModerations([]);
     }
-  }, [pageNumber, moderationStatus, pageSize, processing]);
+  }, [selectedPageNumber, moderationStatus, selectedPageSize, processing]);
```

Comment:
Why is all this stuff in the dependency array? Also can't we just do this next to the `setProcessing(false)` call that would trigger this? Or is the idea that the selected moderations have to be reset when the `selectedPageNumber` and `selectedPageSize` change? Because in that case let's just do it in `handleOnPageNumberChange` and `handleOnPageSizeChange`

Reply:
Yep, you are right, this was completely unnecessary 👍